pkg/commands/util/init_test.go

```diff
@@ -0,0 +1,67 @@
+package util
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
+	crdv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+var (
+	scheme *runtime.Scheme
+)
+
+func init() {
```

Comment:
we should not use `init` in test package

Reply:
😅 I refer to the implementation here and keep it consistent.
I will remove it.

https://github.com/oam-dev/kubevela/blob/64d77656cf5159ecc72493710643de76461c95f7/pkg/controller/dependency/install_test.go#L46-L51

Reply:
yes, we should remove both of them. Thanks for pointing out.